devices/xiaomi.js

```diff
@@ -54,9 +54,15 @@ module.exports = [
         model: 'MCCGQ13LM',
         vendor: 'Xiaomi',
         description: 'Aqara P1 door & window contact sensor',
-        fromZigbee: [fz.ias_contact_alarm_1, fz.aqara_opple],
```

Comment:
Why remove `aqara_opple`?

Reply:
Just copy paste from another converter. Let me check again if it works with aqara_opple.

Reply:
I have two Zigbee coordinators. Main one is CC2538 with firmware 20201010 and one for tests is CC2652P (SONOFF) with firmware 20220103.  Both used with Zigbee2mqtt stable version 1.23.0-1 (different hosts). And I have different responses from Aqara P1 door & window contact sensor.

**[fz.ias_contact_alarm_1, fz.aqara_opple] (CC2652P) :** 
```
Zigbee2MQTT:debug 2022-02-12 13:05:33: Received Zigbee message from '0x54ef4410001e882b', type 'readResponse', cluster 'genPowerCfg', data '{"batteryVoltage":32}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2022-02-12 13:05:33: No converter available for 'MCCGQ13LM' with cluster 'genPowerCfg' and type 'readResponse' and data '{"batteryVoltage":32}'
```
_No Voltage and no contact_

**[fz.ias_contact_alarm_1, fz.battery] (CC2652P) :** 
```
Zigbee2MQTT:debug 2022-02-12 13:23:00: No converter available for 'MCCGQ13LM' with cluster 'genOnOff' and type 'attributeReport' and data '{"onOff":1}'
Zigbee2MQTT:info  2022-02-12 13:23:00: MQTT publish: topic 'zigbee2mqtt/0x54ef4410001e882b', payload '{"battery":100,"contact":null,"last_seen":"2022-02-12T10:23:00.355Z","linkquality":131,"voltage":3200}'
2022-02-12T10:23:00.375Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext []
```
_Got Voltage information, but no contact._

**[fz.ias_contact_alarm_1, fz.aqara_opple] (CC2538) :** 
`Zigbee2MQTT:info  2022-02-12 13:36:08: MQTT publish: topic 'zigbee2mqtt/0x54ef4410001e882b', payload '{"battery":null,"battery_low":false,"contact":true,"last_seen":"2022-02-12T10:36:08.128Z","linkquality":76,"tamper":false,"voltage":null}'`
_No Voltage, but get contact information._ 

**[fz.ias_contact_alarm_1, fz.battery] (CC2538):** 
`Zigbee2MQTT:info  2022-02-12 13:42:52: MQTT publish: topic 'zigbee2mqtt/0x54ef4410001e882b', payload '{"battery":100,"battery_low":false,"contact":false,"last_seen":"2022-02-12T10:42:52.090Z","linkquality":110,"tamper":false,"voltage":3300}'`
_Looks like voltage and contact information available._

Reply:
The cc2538 fw is outdated, it doesn't contain the xiaomi fix needed to support al features for newer xiaomi devices.

CC2652P + `fromZigbee: [fz. xiaomi_contact, fz.ias_contact_alarm_1, fz.aqara_opple, fz.battery],` should do it. Also `voltageToPercentage` should be `3V_2850_3200`